nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/registry/flow/mapping/NiFiRegistryFlowMapper.java

```diff
@@ -193,7 +193,7 @@ public InstantiatedVersionedProcessGroup mapProcessGroup(final ProcessGroup grou
 
     // This is specific for the {@code NifiRegistryFlowRegistryClient}, purely for backward compatibility
     private String getRegistryUrl(final FlowRegistryClientNode registry) {
-        return registry.getComponentType().equals("org.apache.nifi.registry.flow.NifiRegistryFlowRegistryClient") ? registry.getRawPropertyValue(registry.getPropertyDescriptor("URL")) : "";
+        return registry.getComponentType().equals("org.apache.nifi.registry.flow.NifiRegistryFlowRegistryClient") ? registry.getRawPropertyValue(registry.getPropertyDescriptor("url")) : "";
```

Comment:
Suggest adding a static final string variable to define this, eg `private static final String REGISTRY_URL_DESCRIPTOR_NAME = "url";`

Reply:
Hello @thenatog ,
Thank you for your review. I thought add a constant for url but they are not in the same package. Where should i add it? Can you help me?

Reply:
Hello, I added new constant in this class.